### PR TITLE
chore: add treeview templates

### PR DESCRIPTION
**Pull request by inikolova**
Add common templates for the TreeView component and simplify tests

### CODE REVIEW

File: packages/html/src/treeview/templates/treeview-checkboxes.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Treeview, TreeviewItem, TreeviewGroup } from "../../treeview";
+
+export const TreeviewCheckboxes = (props) => (
+    <Treeview
+        children={[
+            <TreeviewItem text="Root 1" showCheckbox />,
```
Comment (Juveniel) [inline packages/html/src/treeview/templates/treeview-checkboxes.tsx:6]:
```suggestion
            <TreeviewItem text="Root 1" showCheckbox />
```

File: packages/html/src/treeview/templates/treeview-icons.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { Treeview, TreeviewItem, TreeviewGroup } from "../../treeview";
+
+export const TreeviewIcons = (props) => (
+    <Treeview
+        children={[
+            <TreeviewItem text="Root 1" />,
```
Comment (Juveniel) [inline packages/html/src/treeview/templates/treeview-icons.tsx:6]:
```suggestion
            <TreeviewItem text="Root 1" />
```

File: tests/_output/bootstrap/treeview/treeview.png
Comment (Juveniel) [inline tests/_output/bootstrap/treeview/treeview.png:1]: the root item in RTL scenarios is not spaced properly, as there needs to be a checkbox there

**Review by Juveniel — CHANGES_REQUESTED** [review]

File: packages/html/src/treeview/templates/treeview-checkboxes.tsx (hunk 1 above)
Comment (inikolova) [reply packages/html/src/treeview/templates/treeview-checkboxes.tsx:6]: The _TreeviewItem_ are defined as children, so I think we need the comma

File: packages/html/src/treeview/templates/treeview-icons.tsx (hunk 2 above)
Comment (inikolova) [reply packages/html/src/treeview/templates/treeview-icons.tsx:6]: The _TreeviewItem_ are defined as children, so I think we need the comma

**Pull request merged**